downstream/modules/hub/proc-deploying-your-system-for-container-signing.adoc

```diff
@@ -58,14 +58,14 @@ automationhub_container_signing_service_key = /absolute/path/to/key/to/sign
 automationhub_container_signing_service_script = /absolute/path/to/script/that/signs
 -----
 +
+//[ddacosta] For 2.5 this will be Log in to Ansible Automation Platform and select Automation Content. Automation hub opens in a new tab. From the navigation ...
 . Once installation is complete, navigate to your {HubName}.
 
-. From the navigation panel, select menu:Signature Keys[].
+. From the navigation panel, select {MenuACAdminSignatureKeys}.
```

Comment:
This one still shows as menu:Signature Keys.

Reply:
@hherbly Yes, I fixed this along with Task Management. Will produce new output for you to verify shortly.